### PR TITLE
NXP-28809: replace label strategy for retention vocabulary

### PR DESCRIPTION
**Pull request by andreacornaglia**

### CODE REVIEW

**Review by pddgoncalves — DISMISSED** [review]

File: nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-create-layout.html [hunk 1]
```diff
@@ -60,7 +60,6 @@
       name="subjects"
       value="{{document.properties.dc:subjects}}"
       multiple="true"
-      dbl10n="true"
```
Comment (guirenard) [inline nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-create-layout.html]: `dc:subjects` is still a dbl10n

File: nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-create-layout.html [hunk 2]
```diff
@@ -72,7 +71,6 @@
       directory-name="l10ncoverage"
       name="coverage"
       value="{{document.properties.dc:coverage}}"
-      dbl10n="true"
```
Comment (guirenard) [inline nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-create-layout.html]: `l10ncoverage` is still dbl10n

File: nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-create-layout.html (hunk 1 above)
Comment (andreacornaglia) [reply nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-create-layout.html]: done

File: nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-create-layout.html (hunk 2 above)
Comment (andreacornaglia) [reply nuxeo-retention-web/document/retentionrules/nuxeo-retentionrules-create-layout.html]: done

**Review by guirenard — DISMISSED** [review]

**Review by pddgoncalves — APPROVED** [review]

**Review by guirenard — APPROVED** [review]

**Pull request merged**